testhelper/docker/resource/minio/minio.go

```diff
@@ -116,3 +126,46 @@ func Setup(pool *dockertest.Pool, d resource.Cleaner, opts ...func(*Config)) (*R
 		Client:          client,
 	}, nil
 }
+
+func (r *Resource) Contents(ctx context.Context, prefix string) ([]File, error) {
```

Comment:
[nitpick] This receiver has a different name than the other one. We could name both `r`.